### DISCUSSION TITLE
Normalization method for scRNA-seq integration and label transfer

### DISCUSSION THREAD

**Discussion by zrcjessica** [stuart-lab/signac#678] [Q&A] [answered]
Hi,

I have a harmony integrated scATAC-seq Signac object as well as an integrated scRNA-seq Seurat object that was generated using SCTransform normalization + RPCA. However, the predicted IDs from the label transfer are a bit surprising to me. I was wondering if I might be doing something wrong - I normalized my gene activity matrix following the [Signac vignettes](https://satijalab.org/signac/articles/pbmc_vignette.html#create-a-gene-activity-matrix-1), using log normalization. However, do I need to normalize my gene activity matrix the same way I normalized my scRNA-seq dataset? So in my case, with SCTransform? 

Thanks!

**Comment by timoast** [comment, marked as answer]
Hi @zrcjessica, you should use the same normalization for both the reference and query datasets when doing integration/transfer. Since we can't run SCTransform on gene activities, you'll need to log-normalize the reference dataset before integrating with scATAC-seq gene activity data.

**Reply by zrcjessica** [reply]
Thanks for the clarification! One more question - what should I set as the Default Assay for my scRNA-seq when I do the label transfer? My understanding is that Log Normalization is meant to be applied to the RNA assay, but since my scRNA-seq is an integrated dataset generated with RPCA + SCTransform, is that still appropriate? Can I or should I perform Log Normalization on the data in the integrated or SCT assay? If so, which data slot?

Thanks!

**Reply by timoast** [reply]
If you want to use integrated scRNA data, you will need to re-run `IntegrateData()` correcting the log-normalized assay rather than the SCT assay. You can then use the log-normalized integrated assay for label transfer with the scATAC-seq data

**Reply by zrcjessica** [reply]
Thanks for describing. I'd like to clarify to make sure I understand what you're saying: are you saying that I need to run `NormalizeData()` and `ScaleData()` on the raw counts from each individual sample in my integrated dataset (e.g. by subsetting my integrated scRNA-seq by sample and then running these functions), and then running `FindIntegrationAnchors()` and `IntegrateData()` on the list of all the log normalized raw counts from each sample in my dataset? And then use this for label transfer? 

Thanks!

**Reply by timoast** [reply]
Yes that's right

**Reply by zrcjessica** [reply]
Sorry, I have another question - should we run `ScaleData()` following `NormalizeData()` on each individual sample, and then run `ScaleData` again on the integrated assay before label transfer?

**Reply by timoast** [reply]
Yes, because you'll need to scale the integrated data to run CCA in the anchor-finding step

**Reply by zrcjessica** [reply]
Thanks so much for the clarification! This helps a lot!

**Reply by zrcjessica** [reply]
Hi Tim,

Just revisiting this question. Earlier you said that I should run `ScaleData()` on each individual sample before integration and then run `ScaleData()` again on the integrated object; however, in [this](https://satijalab.org/seurat/articles/integration_introduction.html) vignette you don't see to run `ScaleData()` on the individual samples before integration. You also don't run `ScaleData()` before finding integration anchors. Which way would be the right way to approach this? Thanks!

**Reply by timoast** [reply]
The `FindIntegrationAnchors()` function scales data for you by default (if `scale=TRUE`), so you don't need to run `ScaleData()` yourself beforehand and it will still work

**Reply by zrcjessica** [reply]
Got it, thanks!
